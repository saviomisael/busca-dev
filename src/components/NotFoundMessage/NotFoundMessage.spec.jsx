import { NotFoundMessage } from '.';

describe('<NotFoundMessage />', () => {
  it('should matches with snapshot', () => {
    const { container } = global.renderTheme(<NotFoundMessage />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
