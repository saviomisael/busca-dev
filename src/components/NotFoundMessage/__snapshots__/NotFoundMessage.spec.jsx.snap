// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundMessage /> should matches with snapshot 1`] = `
.c0 {
  text-align: center;
  text-transform: uppercase;
  -webkit-filter: brightness(0.8);
  filter: brightness(0.8);
  color: #353535;
  font-size: 8.5rem;
  font-family: 'Ubuntu',sans-serif;
}

.c3 {
  color: #F5F5F5;
  -webkit-filter: brightness(1.1);
  filter: brightness(1.1);
}

.c1 > .c2 {
  -webkit-animation: jaKUeX 1s ease-in-out infinite alternate;
  animation: jaKUeX 1s ease-in-out infinite alternate;
}

.c4 {
  font-size: 2.2rem;
}

<div
  class="c0"
>
  <p
    class="c1"
  >
    Erro 
    <span
      class="c2 c3"
    >
      404
    </span>
  </p>
  <p
    class="c1 c4"
  >
    <span
      class="c2 c3"
    >
      Oops
    </span>
    , conteúdo não encontrado!
  </p>
</div>
`;
